examples/language-modeling/run_mlm_flax.py

```diff
@@ -413,7 +422,7 @@ def generate_batch_splits(samples_idx: jnp.ndarray, batch_size: int) -> jnp.ndar
     if samples_to_remove != 0:
         samples_idx = samples_idx[:-samples_to_remove]
     sections_split = nb_samples // batch_size
-    batch_idx = jnp.split(samples_idx, sections_split)
+    batch_idx = np.split(samples_idx, sections_split)
```

Comment:
why not keep `jnp` here?

Reply:
jax version of `split` is very slow on the first call to the function. For debugging purpose I said to Teven to change to np impl which is significantly faster (still on first call only)

Reply:
As discussed with @mfuntowicz , it's much, much slower than `np`. Since this doesn't need to be traced, we don't need to use `jnp`